### PR TITLE
Add Fyld, Landskill and Global Blue to the list

### PR DESCRIPTION
**Pull request by diomonogatari**
Hey there,

Found this and saw that some of the companies I've gone through weren't there.
Decided to contribute :)

Please review and approve.
Cheers,

### CODE REVIEW

**Comment by diomonogatari** [comment]
Hi @marmelo !

Here is my contribution to the list :)
Lemme know if there's anything I've missed.

Cheers,

File: README.md [hunk 1]
```diff
@@ -72,9 +72,11 @@ Checking the company's Tech stack through [Stackshare](https://stackshare.io/) m
 | [DevScope](https://devscope.net/) [:rocket:](https://devscope.net/careers/) | Technology development, consultancy and product development. | `Porto` `Viseu` |
 | [Equal Experts](https://www.equalexperts.com) | Making software. Better. | `Remote` |
 | [Findmore](https://findmore.pt/) [:rocket:](https://findmore.pt/careers) | Nearshoring, software development, UX/UI Design and PMO. | `Lisboa` `Porto` <br> `Viseu` |
+| [Fyld](https://www.fyld.pt) [:rocket:](https://www.fyld.pt/it-jobs) | Consultancy with outsourcing services. Part of Joyn Group companies. | `Lisboa` |
 | [GLanDrive](https://www.glandrive.pt) | Software development and Consultancy. | `Lisboa` `Porto` |
-| [Growin](https://www.growin.com) [:rocket:](https://www.growin.com/it-jobs-growin-careers/) | Software development and system analysis consultancy. | `Lisboa` `Porto` |
+| [Growin](https://www.growin.com) [:rocket:](https://www.growin.com/it-jobs-growin-careers/) | Software development and system analysis consultancy. Part of Joyn Group companies. | `Lisboa` `Porto` |
```
Comment (afonsojramos) [inline README.md:77]:
```suggestion
| [Growin](https://www.growin.com) [:rocket:](https://www.growin.com/it-jobs-growin-careers/) | Software development and system analysis consultancy. | `Lisboa` `Porto` |
```
I'm not sure this is relevant.

**Review by afonsojramos — DISMISSED** [review]
Looking good, thanks! Just a tiny comment for the other maintainers to give their opinion on.

File: README.md (hunk 1 above)
Comment (diomonogatari) [reply README.md:77]: Hey there. Do you want to remove `Part of Joyn Group companies.` as it is too much information?
I've applied it to the 3 companies, Growin, Fyld, and Landskill.

If you guys don't find it relevant, I have no problem removing it :)

Comment (afonsojramos) [reply README.md:77]: I'd say so, but let's wait on others' opinion

Comment (vpereira01) [reply README.md:77]: Agree, "Part of  Joyn Group companies" seems like noise to me.

Comment (diomonogatari) [reply README.md]: @vpereira01 Hey there,

Just removed that info from the companies I added.

Please have a look.
Cheers,

File: README.md [hunk 2]
```diff
@@ -283,6 +285,7 @@ Checking the company's Tech stack through [Stackshare](https://stackshare.io/) m
 | [Fidel](https://fidel.uk) [:rocket:](https://fidel.uk/careers/) | One API for linking bank cards to digital applications globally. | `Lisboa` |
 | [Finiam](https://www.finiam.com/) [:rocket:](https://www.finiam.com/#contact) | Improving people's lives by untangling the financial world. | `Braga` `Remote` |
 | [Fractal](https://web.fractal.id) | Enabling open finance with a new Internet Identity layer. | `Porto` `Remote` |
+| [Global Blue](https://www.globalblue.com/en) [:rocket:](https://www.globalblue.com/en/about-us/career)| Tax-Free Shopping solutions. | `Lisboa` `Porto` `Remote` |
```
Comment (afonsojramos) [inline README.md:288]:
```suggestion
| [Global Blue](https://www.globalblue.com/en) [:rocket:](https://www.globalblue.com/en/about-us/career)| Tax-Free Shopping solutions. | `Lisboa` `Porto` <br> `Remote` |
```

**Review by afonsojramos — CHANGES_REQUESTED** [review]

**Review by afonsojramos — APPROVED** [review]

**Review by vpereira01 — APPROVED** [review]
:+1:

**Pull request merged**